recipe/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
     - numpy
     - scipy
     - nibabel
-    - mne
+    - mne >=0.20
```

Comment:
has this also been updated in the readme of `mne-bids`?

Reply:
No it hasn't.

btw I checked again and the dependency bump is only to make `pytest` happy with our new, stricter approach to also fail at warnings, see https://github.com/mne-tools/mne-bids/pull/376#issuecomment-604496139

Reply:
humm interesting I see. But there is no way to guarantee that I suppose because the test suite is now running on MNE 0.20 instead of 0.19?

Reply:
it seems we _are_ testing with both MNE 0.19 and 0.20. On 0.19, the "0.20-only" tests are apparently being skipped. So instead of depending on 0.20, we should require 0.19.1, like suggested in the mne-bids readme.